js/audio/src/Audio.svelte

```diff
@@ -150,6 +150,7 @@
 			recorder.addEventListener("stop", async () => {
 				recording = false;
 				await dispatch_blob(audio_chunks, "change");
+				await dispatch_blob(audio_chunks, "stop_recording");
```

Comment:
This however is correct, the original implementation was just buggy, this is essentially a similar race condition in that the `stop_recording` event is sent synchronously when the button is clicked while the data being processed is asynchronously and is only complete _after_ the API request is sent.